packages/dotcom-ui-shell/src/components/Shell.tsx

```diff
@@ -32,8 +32,8 @@ type TShellProps = TDocumentHeadProps &
 
 function Shell(props: TShellProps) {
   const bootstrapProps: TBootstrapProps = {
-    coreScripts: [polyfillService.core()],
-    enhancedScripts: [polyfillService.enhanced(), ...props.scripts]
+    coreScripts: [],
+    enhancedScripts: props.scripts
   }
 
   const resourceHints = [
```

Comment:
**question**: it's okay that that there's still a resource hint for polyfill.io, right?

Reply:
let's remove it just in case

Reply:
should be fiiiine seeing as preloading doesn't execute any code but I suppose it would be good to stop fetching it seeing as we don't need it